app/routes/surveys/components/SurveyPage.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React, { Component } from 'react';
+import SurveyList from './SurveyList';
+import styles from './surveys.css';
+import type { SurveyEntity } from 'app/reducers/surveys';
+import { ListNavigation } from '../utils.js';
+import TextInput from 'app/components/Form/TextInput';
+import Content from 'app/components/Layout/Content';
+
+type Props = {
+  surveys: Array<SurveyEntity>,
+  fetching: boolean,
+  addSurvey: SurveyEntity => Promise<*>,
+  deleteSurvey: number => Promise<*>,
+  push: string => void
+};
+
+type State = {
+  searchQuery: string
+};
+
+export default class SurveyPage extends Component<Props, State> {
+  state = {
+    filters: {},
+    searchQuery: ''
+  };
+
+  surveySearch = (surveys: Array<Object>) =>
+    surveys.filter(
+      survey =>
+        survey.title
+          .toLowerCase()
+          .includes(this.state.searchQuery.toLowerCase()) ||
+        survey.event.title
+          .toLowerCase()
+          .includes(this.state.searchQuery.toLowerCase())
+    );
+
+  filterSurveys = (surveys: Array<Object>) =>
+    this.state.searchQuery !== '' ? this.surveySearch(surveys) : surveys;
+
+  updateSearchQuery = (event: Object) => {
```

Comment:
Use `Event` instead of Object

Reply:
This is an html onChange event, not "Arrangement".

Reply:
https://github.com/facebook/flow/blob/master/lib/dom.js#L210

Reply:
How do you import that? 😅

Reply:
Should just work out of the box

Reply:
The DOM event identifiers should be global, you can find the correct one to use here: https://www.saltycrane.com/flow-type-cheat-sheet/latest/#lib/dom.js

Reply:
if it's a synthetic react event, https://www.saltycrane.com/flow-type-cheat-sheet/latest/#lib/react-dom.js

Reply:
I changed it to just `Event`, not feeling to confident about what exactly SyntheticEvents are and when to use them. This `event` is the result of an `onChange` for an input field, for the record.